provider/repodata/data/json/personnames/personnames@1/en.json

```diff
@@ -0,0 +1,270 @@
+{
+  "surname_first_locales": [
```

Comment:
thought: this data can likely be split into a separate key for better deduplication

perhaps combined with given_first_locales. i don't know what to call it though, thoughts @sffc @robertbastian